src/common/BigMovieTile/styled.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export const BigMovieTileWrapper = styled.div`
+  gap: 40px;
+  background: ${({ theme }) => theme.color.black};
+  margin: 0 auto;
+`;
```

Comment:
Problem of Pleksa has to be discussed on next meeting - I will add this task to our board in Trello.